src/app/components/signin/signin.component.css

```diff
@@ -0,0 +1,40 @@
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+}
+input{
+    outline:none;
+    border:none;
+    /* border-radius: 10px; */
+    border-bottom:2px solid gray;
+    width: min(80svw, 900px);
+    /* height: 1rem; */
+    /* width: 100%; */
+    height: 2rem;
+    
+
+}
+form{
+    /* border:2px solid #fff; */
+    /* width:50%; */
+    margin: auto;
+    padding: 1.5rem;
+    box-sizing: border-box;
+    box-shadow : 0 0 5px rgba(0, 0, 0, 0.2)
+
+    
+}
+button{
+    width: min(80svw,900px);
+    background-color: #832BCE;
```

Comment:
Use root variables instead of copying
`background-color: var(--theme-purple-color)`